pyproject.toml

```diff
@@ -192,3 +194,19 @@ strict = true
 [tool.codespell]
 skip = ".git,*.pdf,*.svg"
 ignore-words-list = "theis,coo,homogenous"
+
+[tool.towncrier]
+package = "anndata"
+directory = "docs/release-notes"
+filename = "docs/release-notes/{version}.md"
+single_file = false
+package_dir = "src"
+issue_format = "{{issue}}`{issue}`"
```

Comment:
This is a bit redundant since our current PR system has the `{pr}` or `{issue}` tag at the end.  I am going to update the fragments here.